CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The format of this document is based on [Keep a Changelog](https://keepachangelo
 ### Added
 - Add the usage of `MultipleAnalogInterfaces`(https://www.yarp.it/group__dev__iface__multiple__analog.html)
   in the `gazebo_yarp_imu`.
+- Add the possibility to create different types of joints with the `linkattacher` plugin (https://github.com/robotology/gazebo-yarp-plugins/pull/461).
```

Comment:
wrong location, should move it among unreleased

Reply:
done https://github.com/robotology/gazebo-yarp-plugins/pull/461/commits/6917e052926ff0020873cfba94c756248d0c4b84